apps/storefront/package.json

```diff
@@ -47,7 +53,8 @@
     "react-intl": "6.2.1",
     "react-use": "17.4.0",
     "tailwind-scrollbar-hide": "1.1.7",
-    "url-join": "4.0.1"
+    "url-join": "4.0.1",
+    "urql": "^3.0.3"
```

Comment:
At the moment, SDK examples and app-templates use urql. Storefront uses Apollo. How should we approach? We can adapt to Apollo, but maybe there is some migration plan to urql (its sed in checkout?)

Its also backend-only code, so shouldn't affect bundle

Reply:
I'm not sure if there's a migration plan. I don't have any preference for either urql or apollo, so we should just pick one and plan the migration.